lib/challenge.js

```diff
@@ -1,5 +1,33 @@
 'use strict';
 
+let milli = 1000000;
+let sumTen = 0;
+let sumTwo = 0;
+
+String.prototype.reverse = function() {
+	return this.split('').reverse().join('');
+};
+
+let baseTen = function() {
+  for (let i =0; i < milli; i++) {
+    let iToString = i.toString();
+    if ( iToString === iToString.reverse() && (i % 10) === i) {
+      sumTen = sumTen + i;
+    }
+  }
+  return sumTen;
+}
+
+let baseTwo = function () {
+  for (let i =0; i < milli; i++) {
+    let iToString = i.toString();
+    if ( iToString === iToString.reverse() && (i % 2)  === i) {
```

Comment:
Ah, I see - you're trying to implement the base10/base2 thing. 

There's a simple way to do that in JS - when you call `.toString` on a number, you can specify a radix (i.e. base) to represent the number in.

```javascript
> 3.toString(2)
'11'
```